nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_36PB/metadata.yaml

```diff
@@ -56,36 +56,37 @@ implemented_observables:
       - $W^-$
       - $W^-$
       - $W^-$
-    plot_x: k1
+    plot_x: abs_eta
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - abs_eta
+  - m_W2
+  - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
+      abs_eta:
+        description: Absolute pseudo-rapidity of the W boson
+        label: abs_eta
         units: ''
-    file: kinematics_ETA.yaml
+      m_W2:
+        description: Squared invariant mass of the W boson
+        label: m_W2
+        units: 'GeV^2'
+      sqrts:
+        description: Square root of the center of mass energy
+        label: sqrts
+        units: 'GeV'
+    file: kinematics.yaml
   theory:
-    conversion_factor: 1.0
+    conversion_factor: 1.0187 #is this needed?
```

Comment:
I am not sure if I should include this conversion factor here

Reply:
looking at commondataparser.py this is valid input! So I would keep it.